README.md

```diff
@@ -102,10 +114,21 @@ This extension provides 4 actions:
     ```php
     <?php
 
+    namespace app\controllers;
+
+    use Yii;
+    use yii\web\Controller;
+    use yii\helpers\Url;
+
+
     class SamlController extends Controller {
 
+        // Remove CSRF protection
+        public $enableCsrfValidation = false;
```

Comment:
i think it would be better just disable CSRF for only ACS action, so maybe something like this

````php
    public function beforeAction($action)
    {
        if ($action->id == 'acs') {
            $this->enableCsrfValidation = false;
        }
        return true;
    }
````